content/docs/index.md

```diff
@@ -34,9 +34,9 @@ automate tests and deployments using GitOps.
 
 </cards>
 
-✅ Please join our [community](/community) or use the [support](/support)
-channels if you have any questions or need specific help. We are very responsive
-⚡.
+✅ Please join our [community](https://dvc.org/community) or use the
```

Comment:
cml.dev's doc page links to dvc.org as well, but we could also update the text to say something else!